language-extensions/python/build/windows/restore-packages.cmd

```diff
@@ -40,30 +43,52 @@ curl -sS https://bootstrap.pypa.io/get-pip.py |"%PYTHON_INSTALLATION_PATH%\pytho
 
 REM Install numpy and pandas
 REM
-"%PYTHON_INSTALLATION_PATH%\python.exe" -m pip install pandas numpy
+SET NUMPY_VERSION=1.22.3
+SET PANDAS_VERSION=1.4.2
+"%PYTHON_INSTALLATION_PATH%\python.exe" -m pip install --force-reinstall numpy==%NUMPY_VERSION% pandas==%PANDAS_VERSION%
 
 REM Remove the Python installer which is no longer needed
 REM
 del "python-%PYTHON_VERSION%.exe"
 
+REM BOOST artifact download, extract, build
 REM Download the specified version of Boost from SourceForge
 REM Extract the downloaded Boost zip file to the packages directory
 REM Remove the Boost zip file
 REM Navigate to the extracted Boost directory
 REM
-curl -L -o boost_%BOOST_VERSION_IN_UNDERSCORE%.zip https://sourceforge.net/projects/boost/files/boost/%BOOST_VERSION%/boost_%BOOST_VERSION_IN_UNDERSCORE%.zip/download
-powershell -NoProfile -ExecutionPolicy Unrestricted -Command "Expand-Archive -Force -Path 'boost_%BOOST_VERSION_IN_UNDERSCORE%.zip' -DestinationPath '%PACKAGES_ROOT%'"
-del boost_%BOOST_VERSION_IN_UNDERSCORE%.zip
+curl -L -o boost_%BOOST_VERSION_IN_UNDERSCORE%.7z https://archives.boost.io/release/%BOOST_VERSION%/source/boost_%BOOST_VERSION_IN_UNDERSCORE%.7z
+
+REM -o Output directory
+REM 2nd param is the file to expand
+echo -- Beginning Boost ZIP extraction -- Time: %time% --
+%ARCHIVE_TOOL_PATH% x -y -o"%PACKAGES_ROOT%" "boost_%BOOST_VERSION_IN_UNDERSCORE%.7z"
+echo -- Finished Boost Zip extration -- Time: %time% --
+
+REM Boost cleanup
+echo Delete 7z boost archive
+del boost_%BOOST_VERSION_IN_UNDERSCORE%.7z
+echo go to dir with boost unarchived
+echo %cd%
```

Comment:
Does this echo print "change directory" with %cd%?

Reply:
`echo %cd%` prints the current directory.